src/core/core.cpp

```diff
@@ -104,13 +102,19 @@ bool lif::initCore() {
 		return false;
 
 	// Fill the assetDir and saveDir variables once and for all
+#if defined(SFML_SYSTEM_MACOS)
+	// In addition to saved games, saveDir is also used to save preferences
+	assetDir = bundleResourcesPath() + DIRSEP;
+	saveDir = applicationSupportPath() + DIRSEP + "BOOM Remake" + DIRSEP;
```

Comment:
Do we _really_ want a space in the folder path, if we can help it?

Reply:
Other Mac apps with a space in the name also save in ApplicationSupport with the same space in the name.

Reply:
I'll trust you on this since I'm not an expert in Mac conventions.